src/components/manage/Sharing/Sharing.jsx

```diff
@@ -145,6 +145,7 @@ class SharingComponent extends Component {
     this.onToggleInherit = this.onToggleInherit.bind(this);
     this.state = {
       search: '',
+      isLoading: false,
```

Comment:
@tedw87 This should also be used to block the search button or the form re-submit as right now it looks like I am able to DDOS attack the backend RestAPI endpoint by clicking the search button or by hitting the `Enter` key

![Screenshot from 2023-09-07 17-47-32](https://github.com/plone/volto/assets/323385/f70c0294-d0d5-45f1-b19f-e523b657fbd4)